### PR TITLE
SM proto: move gid from instance info to service info

### PR DESCRIPTION
**Pull request by Eabramov84**

### CODE REVIEW

**Review by MykolaSolyanko — APPROVED** [review]

File: aostypes/aostypes.go [hunk 1]
```diff
@@ -128,6 +128,7 @@ type ServiceInfo struct {
 	VersionInfo
 	ID         string `json:"id"`
 	ProviderID string `json:"providerId"`
+	GID        uint32 `json:"gid"`
```
Comment (al1img) [inline aostypes/aostypes.go:131]: Do we need it here? SM can take it itself from service config.

File: api/cloudprotocol/cloudprotocol.go [hunk 2]
```diff
@@ -433,6 +433,7 @@ type PartitionInfo struct {
 // NodeInfo node information.
 type NodeInfo struct {
 	NodeID     string          `json:"nodeId"`
+	NodeModel  string          `json:"nodeModel"`
```
Comment (al1img) [inline api/cloudprotocol/cloudprotocol.go:436]: `NodeType`

File: aostypes/aostypes.go (hunk 1 above)
Comment (Eabramov84) [reply aostypes/aostypes.go:131]: GID is generated by imagemanager.

File: api/cloudprotocol/cloudprotocol.go (hunk 2 above)
Comment (Eabramov84) [reply api/cloudprotocol/cloudprotocol.go]: added

**Review by al1img — APPROVED** [review]

**Pull request merged**